modules/sdk-lib-mpc/src/tss/ecdsa/primes.ts

```diff
@@ -0,0 +1,1842 @@
+export const primesSmallerThan319567 = [
```

Comment:
Alpha is a constant value. If it ever changes, we should update this file also. 

The reason we hardcoded this is because re-deriving all these prime numbers takes like 15 seconds + when I did my tests.

Reply:
(hardcoding this, the entire test suite finishes in under 7 seconds) 
![image](https://github.com/BitGo/BitGoJS/assets/24881706/dcbea0ed-490d-4a15-bda7-de56d83580f1)

Reply:
>  if we made an external call how long that would take

This would be a security risk, almost all security values should be derived purely client side. The exception is challenge values.